rest_framework/compat.py

```diff
@@ -4,7 +4,7 @@
 """
 
 # flake8: noqa
-from __future__ import unicode_literals
+from __future__ import unicode_literals, division
```

Comment:
I don't think we actually need this - the only division we've added will always be float on both 2 and 3, since we're dividing by a float.